traffic_editor/scripts/generate_thumbnails.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import yaml
+import argparse
+
+
+class ThumbnailsGenerator:
+    def __init__(self, argv=sys.argv):
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+                'models_dir', default='~/.gazebo/models',
+                help='Directory of models to be added')
+        parser.add_argument(
+                'model_list', default='../test/model_list.yaml',
+                help='Path of model_list.yaml')
+        parser.add_argument(
+                'output_dir',
+                help='Directory where the output images will be saved')
+        args = parser.parse_args(argv[1:])
```

Comment:
just a note, that if you manage to add the arg parser in the plugin, you will also be able to propagate the arguments to this script as well